alts/worker/tasks.py

```diff
@@ -100,6 +104,10 @@ def is_success(stage_data_: dict):
         runner.setup()
         runner.install_package(package_name, package_version)
         runner.run_package_integrity_tests(package_name, package_version)
+    except InstallPackageError as exc:
+        logging.exception('Cannot install package: %s', exc)
```

Comment:
passing `exc` to logging is not needed, because `logging.exception` can itself catch info about an exception.

```
>>> import logging
>>> logger = logging.getLogger(__name__)
ing.basi>>> logging.basicConfig(
...     level=logging.INFO,
...     format='%(asctime)s | %(name)s | %(levelname)s | %(message)s',
... )
>>> def test1():
...   try:
...     raise ValueError('Some message')
...   except:
...     logger.exception('Some exception')
...
>>> def test2():
...   try:
...     raise ValueError('Some message')
...   except Exception as err:
...     logger.error('Some exception %s', err)
...
>>> test1()
2021-11-22 14:19:13,528 | __main__ | ERROR | Some exception
Traceback (most recent call last):
  File "<stdin>", line 3, in test1
ValueError: Some message
>>> test2()
2021-11-22 14:19:17,351 | __main__ | ERROR | Some exception Some message
>>>
```